pentaho-gwt-widgets/src/org/pentaho/gwt/widgets/client/colorpicker/TransparencyImpl.java

```diff
@@ -416,7 +416,7 @@ public static void setTransparency( Element elem, int alpha ) {
       DOM.setStyleAttribute( elem, "filter", "alpha(opacity=" + alpha + ")" ); //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
     } else {
       // Everyone else
-      DOM.setStyleAttribute( elem, "-moz-opacity", "" + ( new Integer( alpha ).floatValue() / 100 ) + "" ); //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
+      DOM.setStyleAttribute( elem, "MozOpacity", "" + ( new Integer( alpha ).floatValue() / 100 ) + "" ); //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
```

Comment:
@krivera-pentaho Kris, is this OK to change? I know this one changed through versions. We need to make sure that we're not breaking something here. Thanks!